webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,20 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/<user_api_key>/get_pending_jobs", methods=["GET"])
```

Comment:
We decided that we would use the `core` blueprint only for lowlevel submission and retrieval. We can make a new blueprint for dataset evaluation items, since eventually you will need a number of endpoints for this project.